try {
 ...
} catch (IOException e) {
    throw new RuntimeException(e);
}